common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -102,7 +102,61 @@ def test_clarification(self):
         self.assertNotIn('strong', tooltip_text)
 
 
-class ProblemExtendedHintTest(ProblemsTest, EventsTestMixin):
+class ProblemHintTest(ProblemsTest, EventsTestMixin):
```

Comment:
Note to reviewers: created a base test class for two hint-related classes that basically just had duplicated code.